gflib/malloc.h

```diff
@@ -11,6 +10,8 @@
 
 #define TRY_FREE_AND_SET_NULL(ptr) if (ptr != NULL) FREE_AND_SET_NULL(ptr)
 
+// 122 KB. Max size of the heap without running into other data
```

Comment:
I don't think having this comment makes sense? The max size of the heap should just be its current size + however many bytes remain free in EWRAM, which is true of everything in EWRAM (and if we're printing the number of free bytes it makes less sense to list a static number here). Plus users would be alerted if it's oversized when it fails to link.

Unrelated but if you want you can update that declaration to `extern u8 gHeap[HEAP_SIZE];` so people can use `sizeof` if they want.

Reply:
In the base game the heap doesn't consume all remaining EWRAM and it was placed at the front of the rom. 
Additionally whenever the heap is initialised it's passed the max size of that instance as a parameter (in practice only HEAP_SIZE). e.g.: https://github.com/pret/pokeemerald/blob/b302291bd3ef433c6eeed9a6940036d527ab13ae/src/credits.c#L417

So my thinking was that the constant HEAP_SIZE was in effect the *maximum heap size* and not technically the true heap size at any given moment.

Probably could convey that better but for now I'll remove it.

Reply:
Yeah I think it was just a miscommunication, I interpreted the comment to mean "this is the maximum value `HEAP_SIZE` can be"